### ISSUE TITLE
Extract algorithm for MVP challenge 

### ISSUE THREAD

**Issue by Maartenvm** [EYRA-Benchmark/project-backlog#28]
Extract algorithm code from Eveline's notebook into python files and convert to Docker algorithm container

**Comment by Maartenvm** [comment]
One done (B). The other two algorithms (A and C), need a split of the training set in a validation set.

**Comment by Maartenvm** [comment]
A and C done. B needs re-doing because of renewed cookiecutter template
